package.json

```diff
@@ -8,17 +8,20 @@
     "prebuild": "rimraf dist",
     "build": "nest build",
     "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
+    "clean": "rm -rf node_modules && yarn && yarn upgrade",
     "start": "node dist/main",
```

Comment:
@kelvin2go Should this one be the same as `start:prod`?

`yarn start` doesn't seem to work anymore as said in the Readme, probably needs some update.